tests/client/test_3_lvls_stack_simple_calls.txt

```diff
@@ -0,0 +1,30 @@
+7
```

Comment:
I'm not sure what's the format of this file. Could you somehow document it?

Reply:
I've wrote in the PR's description what the format of the file is. However, after reading your comments, maybe it's not the best format.